demo/api/src/content-generation.module.ts

```diff
@@ -0,0 +1,84 @@
+import { ContentGenerationRequest, Options } from "@comet/cms-api/lib/content-generation/content-generation.types";
+
+export const ContentGenerationModels = ({ apiKey, apiUrl }: { apiKey: string; apiUrl: string }) => {
```

Comment:
This is an example implementation for the Azure Image Segmentation API, am I right? We should add this somewhere in the code so it's clear what service this example is targeting. Maybe in the name of the env variable (?)

Reply:
Maybe we should rename the file and everything in it into Azure? Like `azure-content-generation.ts`?

Reply:
> Maybe we should rename the file and everything in it into Azure? Like `azure-content-generation.ts`?

Would be fine by me

Reply:
May be add an AzureContentGenerationModule, that imports and configures the ContentGenerationModule? That would be more Nest-like IMO